lib/runner.js

```diff
@@ -419,20 +419,6 @@ Runner.prototype.fail = function(test, err, force) {
  * @param {Error} err
  */
 Runner.prototype.failHook = function(hook, err) {
```

Comment:
could probably just remove `failHook` and replace its usage with calls directly to `fail`

Reply:
Will do.